concent_api/core/payments/backends/sci_backend.py

```diff
@@ -28,15 +30,15 @@ def get_list_of_payments(
 
     if transaction_type == TransactionType.FORCE:
         payments_list = concent_rpc.get_forced_payments(  # pylint: disable=no-member
-            requestor_address   = requestor_eth_address,
-            provider_address    = provider_eth_address,
+            requestor_address   = Web3.toChecksumAddress(requestor_eth_address),
+            provider_address    = Web3.toChecksumAddress(provider_eth_address),
```

Comment:
Wouldn't it be better to let `get_forced_payments()` do the address conversion internally? It looks like an implementation detail to me.

Reply:
But, we need to do this conversion before we give addresses as parameters. If not some exception is raised.

Reply:
But `get_force_payments()` is our function. If it raises an exception then maybe it shouldn't?